### DISCUSSION TITLE
[1.8.4] Brook X One adapter does not work in Switch and PS4 mode

### DISCUSSION THREAD

**Discussion by Invictaz** [darthcloud/BlueRetro#749] [Q&A]
Connected the Brook X one Adapter to my Xbox One 1708 controller. 
Brook adapter set to Nintendo Switch Pro controller mode. 
Does not want to pair with Blueretro PSUnoRetro Adapter.

HCI log captured via ubuntu 

Nintendo Switch setting

[hcitrace.zip](https://github.com/darthcloud/BlueRetro/files/12387004/hcitrace.zip)

[~hcitrace.txt](https://github.com/darthcloud/BlueRetro/files/12387002/hcitrace.txt)

and in PS4 mode it does connect and the PSUnoRetro light goes off, but there is no button input.

[~hcitrace2.txt](https://github.com/darthcloud/BlueRetro/files/12387011/hcitrace2.txt)
[hcitrace2.zip](https://github.com/darthcloud/BlueRetro/files/12387012/hcitrace2.zip)

**Comment by darthcloud** [comment]
Supporting stuff that try to mock another thing is very hard as I can't identify it vs the real thing.

Can you try in PC mode instead? View + RB as per manual.

Take a trace in that mode and press every button and take note of the order you pressed them. 

Upload trace here and also the buttons press order list.

**Reply by Invictaz** [reply]
Here it is in PC mode. @darthcloud 

Button order

Dpad:

Up 
down
left
right

A
B
X
Y

View
Menu
Xbox button

LB
RB
LT
RT

L3
R3

Analog sticks are seperate
Left stick

up 
down
left

right

Right stick

Up
down
left
right

[hcitrace.zip](https://github.com/darthcloud/BlueRetro/files/14380809/hcitrace.zip)
[hcitrace-analog.zip](https://github.com/darthcloud/BlueRetro/files/14380848/hcitrace-analog.zip)

![Screenshot from 2024-02-23 01-23-33](https://github.com/darthcloud/BlueRetro/assets/20338545/28b9f843-125c-43fb-b9cd-b97fd19261b7)

**Comment by darthcloud** [comment]
I opened an issue to track this #961 . I will forgot otherwise.

**Reply by Invictaz** [reply]
Thanks. I will keep track on the issue tracker
